prow/jobs/kyma/kyma-integration-gardener.yaml

```diff
@@ -0,0 +1,187 @@
+        env:
+          - name: KYMA_PROJECT_DIR
+            value: "/home/prow/go/src/github.com/kyma-project"
+          - name: GARDENER_REGION
+            value: "europe-west4"
+          - name: GARDENER_ZONES
+            value: "europe-west4-b"
+        resources:
+          requests:
+            memory: 1Gi
+            cpu: 400m
+
+gardener_azure_job_labels_template: &gardener_azure_job_labels_template
+  preset-gardener-azure-kyma-integration: "true"
+  preset-kyma-cli-stable: "true"
+  preset-creds-aks-kyma-integration: "true"
+
+gardener_azure_job_template: &gardener_azure_job_template
+  skip_report: false
+  decorate: true
+  path_alias: github.com/kyma-project/kyma
+  max_concurrency: 10
+  spec:
+    containers:
+      - <<: *base_image_115
+        securityContext:
+          privileged: true
+        command:
+          - "bash"
+        args:
+          - "-c"
+          - "${KYMA_PROJECT_DIR}/test-infra/prow/scripts/cluster-integration/kyma-integration-gardener-azure.sh"
+        env:
+          - name: RS_GROUP
+            value: "kyma-gardener-azure"
+          - name: REGION
+            value: "northeurope"
+          - name: EVENTHUB_NAMESPACE_NAME
+            value: "kyma-gardener-azure"
+          - name: KYMA_PROJECT_DIR
+            value: "/home/prow/go/src/github.com/kyma-project"
+          - name: GARDENER_REGION
+            value: "westeurope"
+          - name: GARDENER_ZONES
+            value: "1"
+        resources:
+          requests:
+            memory: 1Gi
+            cpu: 400m
+
+presubmits:
+  kyma-project/kyma:
+    - name: pre-master-kyma-gardener-azure-integration
+      <<: *gardener_azure_job_template
+      optional: true
+      branches:
+        - ^master$
+      # following regexp won't start build if only Markdown files were changed
+      run_if_changed: "^(((resources/knative-eventing|resources/knative-eventing-kafka|resources/knative-serving|resources/application-connector)\\S+)(\\.[^.][^.][^.]+$|\\.[^.][^dD]$|\\.[^mM][^.]$|\\.[^.]$|/[^.]+$))"
+      labels:
+        preset-build-pr: "true"
+        <<: *gardener_azure_job_labels_template
+      extra_refs:
+        - <<: *test_infra_ref
+          base_ref: master
+
+postsubmits:
+  kyma-project/kyma: []
+#    - name: post-master-kyma-gardener-azure-integration
+#      <<: *gardener_azure_job_template
+#      optional: true
+#      annotations:
+#        testgrid-dashboards: kyma-postsubmit
+#        # testgrid-alert-email: email-here@sap.com
+#        testgrid-num-failures-to-alert: '1'
+#      branches:
+#        - ^master$
+#      labels:
+#        preset-build-master: "true"
+#        <<: *gardener_azure_job_labels_template
+#      extra_refs:
+#        - <<: *test_infra_ref
+#          base_ref: master
+
+periodics:
+  - name: kyma-integration-gardener-gcp
+    <<: *gardener_gcp_job_template
+    decorate: true
+    decoration_config:
+      timeout: 14400000000000 # 4h
+      grace_period: 600000000000 # 10min
```

Comment:
(nit) Could these values be like:
```
timeout: 4h
grace_period: 10m
```

If yes, then please apply to the rest of the template.